p2p/lightchain.py

```diff
@@ -202,8 +213,8 @@ def peer_block_height(peer: LESPeer) -> int:
                 raise LESAnnouncementProcessingError(
                     "No common ancestors found between us and {}".format(peer))
             except TooManyTimeouts:
-                raise LESAnnouncementProcessingError(
-                    "Too many timeouts when fetching headers from {}".format(peer))
+                self.logger.debug("Too many timeouts when getting common ancestors from %s", peer)
```

Comment:
I don't see the point of the debug log here given that you let the exception bubble up and then log a warning with it further up.

Reply:
Mostly so we could see the context of what it was working on when it timed out. The warning log will only have whatever message got raised during `TooManyTimeouts`. I'm pretty neutral on this, though, so if you still think it's not helpful info, I'm okay to drop. :+1: